src/DbTests.py

```diff
@@ -10,7 +10,7 @@
 import discord as dis
 import json
 import sys
-import mariadb
+#import mariadb
```

Comment:
Not needed anymore?

Reply:
Removed.  But yeah, I got the mock working correctly now, though actually mocking the interface is going to be a nightmare.  The `Cursor's` `execute()` function doesn't take arguments well, so determining how to mock them based on the command will be strenuous.   For now I'm punting the DB unit tests, but it would be nice to actually enable them (I caught an error in integration testing earlier).

Reply:
Okay not needed then, could remove the comment whenever then